### DISCUSSION TITLE
part-ii/2006-18/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#211] [Question comments]
Part II 2006 Coding and Cryptography: 1.I.4G 

https://questions.tripos.org/part-ii/2006-18/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork. I won't type up the working for the Berlekamp-Massey method, but we get that

$$x_n = x_{n-1} + x_{n-3} + x_{n-5}$$

</details>
